name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java

```diff
@@ -347,17 +347,18 @@ public static ChartRange createFor(List<SecurityPrice> prices, ChartInterval cha
 
     private Color colorQuote;
 
-    private static final Color colorEventPurchase = Colors.getColor(26, 173, 33);
-    private static final Color colorEventSale = Colors.getColor(232, 51, 69);
-    private static final Color colorEventDividend = Colors.getColor(128, 0, 128);
+    private static final Color colorEventPurchase = Colors.getColor(26, 173, 33); // #1AAD21
+    private static final Color colorEventSale = Colors.getColor(255, 43, 48); // #FF2B30
+    private static final Color colorEventDividend = Colors.getColor(191, 62, 255); // #BF3EFF
```

Comment:
In my personal opinion the color is too bright for the dark background and therefore the numbers are harder to read. So slightly darker than your current proposal would be nice. Currently I have not time to test with some colors around.

Reply:
Adjusted